qe-tests/cypress/integration/models/plan.ts

```diff
@@ -145,6 +146,29 @@ export class Plan {
       });
   }
 
+  protected waitForCanceled(name: string): void {
+    //Go to Migration plans list page
+    Plan.openList();
+    cy.get(tdTag)
+      .contains(name)
+      .parent(tdTag)
+      .parent(trTag)
+      .within(() => {
+        cy.get(dataLabel.status).contains(planCanceledMessage, { timeout: 3600 * SEC });
+      });
+  }
+
+  protected plan_details(name: string): void {
+    cy.get(tdTag).contains(name).parent(tdTag).parent(trTag).click();
+  }
+
+  protected cancel(name: string): void {
+    this.plan_details(name);
+    cy.get(`[aria-label="Select row 0"]`, { timeout: 20000 }).should('be.enabled').check();
```

Comment:
FYI, this selector may break in the future if we improve the aria-label here to include the row's name (which I think we may end up doing).